index.js

```diff
@@ -1,3 +1,19 @@
+/* Copyright (c) 2010 - 2020, Nordic Semiconductor ASA
```

Comment:
The start year should reflect the date of the oldest content still in the work, so should probably be 2019 or so for all the files. Not super important, thought it does affect when expiry of certain permissions take act.